src/main/java/seedu/address/model/settings/DailyTarget.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.model.settings;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class DailyTarget implements Comparable {
+
+    public static final String MESSAGE_CONSTRAINTS = "Daily Target should not exceed 720 mins";
```

Comment:
How did you arrive at 720?

Reply:
12 x 60. Each day one should not exceed 12 hours of work to prevent burnout. Just a rough estimation of the top of my head!